debug/app/index.html

```diff
@@ -4,5 +4,6 @@ <h1><a href="..">SMK Debug</a> / Apps</h1>
     <li><a href="wf">Wildfirex</a></li>
     <li><a href="translink">Translink</a></li>
     <li><a href="francophone">Francophone</a></li>
+    <li><a href="bespoke">Bespoke Tool</a></li>
```

Comment:
I'm not sure that this 'belongs' in the demo app. The other options here are all custom applications, not just a demonstration of a tool. I'm not sure that the ToolConfig section is proper either. What do you think about either putting it at the top level (on the root SMK Debug page)? Or even having a 'Custom Code' link at the top level and then the Bespoke tool demo nestled in there?

Reply:
I chatted with Nicole about this and she agrees about moving this out of the "app" area. One suggestion we talked about is to move it into the Features/Tool Config area, where code is shown next to an app using the tool.